test/api/response_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule Test.Api.ResponseTest do
+  use ExUnit.Case
+  doctest ExUid2.Api.Response
+
+  alias ExUid2.Api.Response
+
+  test "Responses can be decrypted" do
+    secret_key = :crypto.strong_rand_bytes(32)
+    iv = :crypto.strong_rand_bytes(12)
+
+    message = %{"message" => "test"}
+
+    encoded_message = Jason.encode!(message)
+
+    {:ok, {encrypted_message, tag}} = encrypt_response(encoded_message, iv, secret_key)
+
+    {:ok, decrypted_message} = Response.decrypt(encrypted_message, iv, tag, secret_key)
+
+    assert encoded_message == decrypted_message
+  end
+
+  test "Decrypting and parsing a response returns the right object" do
+    secret_key = :crypto.strong_rand_bytes(32)
+    iv = :crypto.strong_rand_bytes(12)
+
+    message = %{"message" => "test"}
+
+    json_message = Jason.encode!(message)
```

Comment:
This would be a cute place to use `~J`

Reply:
Not quite! `~J` doesn't encode a JSON object, it decodes it. And I'd need to import it first so I'm not even saving characters!
```
iex(1)> import Jason.Sigil
Jason.Sigil
iex(2)> ~J'{"a":"b"}'
%{"a" => "b"}
```